pkg/k8s/pipelineRun_test.go

```diff
@@ -86,8 +86,39 @@ func Test__calling_UpdateState_and_FinishState_yieldsHistoryWithOneEntry(t *test
 	assert.Equal(t, 1, len(status.StateHistory))
 }
 
+func Test_GetBaseRepoURL_CorrectURL(t *testing.T) {
+	factory := fake.NewClientFactory(newPipelineRunWithURL("https://github.com/SAP"))
+	r, _ := NewPipelineRunFetcher(factory).ByName(ns1, run1)
+	url, err := r.GetRepoBaseURL()
+	assert.Equal(t, "https://github.com", url)
+	assert.NilError(t, err)
+}
+
+func Test_GetBaseRepoURL_CorrectURLWithPort(t *testing.T) {
```

Comment:
The test method name should contain the _full_ name of the examinee:

    Test_pipelineRun_GetBaseRepoURL_...